content/_getting-started/first-flow.md

```diff
@@ -1,172 +1,123 @@
-to synchronize the data between two applications or services.
+This document provides a step-by-step instruction on creating a basic [Integration Flow](integration-flow). It is intended for platform users who have little to no experience in Flow creation. The instruction includes the following sections: [configuring trigger](#configuring-trigger), [configuring action steps](#configuring-action-steps) and [post-creation activities](#post-creation-activities).
 
-We assume that this is your first experience with the {{site.data.tenant.name}}
-platform, so we explain every step in detail. Please follow these steps to create
-your first integration flow.
+## Configuring Trigger
 
-## Choosing the data source
+The first step of an Integration Flow is called an initial trigger. Its job is to trigger the running Flow execution. [Components](/integration-component) that have a trigger mode can be used for a trigger, for example: Node.js, Webhook Component, Simple Trigger, etc. For the first step, the list of available components is limited to those that have trigger mode. In this instruction we will use Simple Trigger as an example of the first step.  
 
-When you login first time you will have a similar view of the {{site.data.tenant.name}} platform.
+To start the creation of our Flow, go to the navigational menu, select your Workspace and either click "Add new flow" on the dashboard:
 
-![Start creating your first flow](/assets/img/getting-started/first-flow/getting-started-flow-001.png "Start creating your first flow")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_0.png)
 
-To start click on *+Add new flow* button to load the integration flow designer.
+**OR** go to *Flows* and click "Add New Flow":
 
-The designer helps building the flows from the reusable integration components
-such as `triggers` and `actions`. Every integration flow starts with a `trigger`
-component to retrieve data from the source. To start let us find the
-*Petstore component* in the list of integration components.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_1.png)
 
-![Select our Petstore component](/assets/img/getting-started/first-flow/getting-started-flow-002.png "Select our Petstore component")
+This brings you to *Designer* page. Let's start with naming your Flow, which is optional, and adding the initial trigger:
 
-For that purpose type `petstore` in the search bar and you will find it. Next,
-click on *Choose Petstore API (Node.js)* button.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_4.png)
 
-![Create the credentials to access](/assets/img/getting-started/first-flow/getting-started-flow-003.png "Create the credentials to access")
+The platform prompts you to setup the first step. First you may give the step a special name. Then you can select the Component you want to use, by using search bar, Component list, or load new Component from Catalog:
```

Comment:
While reading
> First you may give the step a special name

I was not sure what screenshot are you talking above. I was assuming you are talking about the following screenshot but you are talking about the screenshot above. Maybe we should explicitly mention something like this

> In the screenshot above you can see ....

Plus we could place some number on the arrows in the screenshots and use the same numbers in the text. For example:

> The platform prompts you to setup the first step. First you may give the step a special name (1). Then ...

Reply:
Also see my comment below regarding `load new Component from Catalog`

Reply:
This is strange. I was talking about the screenshot below.

Also, I'm a bit prejudiced to text/numbers in screenshots. I'm going to make a split-test among our guys and see what they think about it.